baselines/train_ec2_excessive_runinstances.yml

```diff
@@ -0,0 +1,57 @@
+baseline:
+  splunk:
+    macros:
+      - ec2_excessive_runinstances_mltk_input_filter
+    schedule:
+      cron_schedule: ''
+      earliest_time: -91d@d
+      latest_time: -1d@d
+    search: sourcetype=aws:cloudtrail eventName=RunInstances errorCode=success | bucket span=10m _time | stats sum(instances_launched) as instances_launched by _time userName | fit DensityFunction instances_launched threshold=0.0005 into ec2_excessive_runinstances_v1
```

Comment:
We need the search syntax to use datamodels and have the macros in it. Similar to 
https://github.com/splunk/security-content/blob/73800355367d8b99ff69a9596b908f3bdda76e52/baselines/baseline_cloud_compute_regions.yml#L11

Reply:
I've added the input filter macro in my last commit. Dorsey mentioned that we could do a separate AWS and generic cloud analytics stories for this.